tests/rbd_mirror/test_rbd_mirror_journal_to_snap.py

```diff
@@ -67,16 +102,56 @@ def run(**kw):
                 imagespec=imagespec,
                 state_pattern="up+replaying",
             )
-
-        # Cleans up the configuration
-        mirror1.clean_up(peercluster=mirror2, pools=[poolname])
+        # Verification of Negative test as image should not allow write operation from secondary
+        test_write_from_secondary(mirror2, mirrormode, imagespec)
```

Comment:
The return value from this method should be returned back to run() method. So I guess you need to write something like

if test_write_from_secondary(mirror2, mirrormode, imagespec):
return 1

Reply:
Done